### DISCUSSION TITLE
Single select option

### DISCUSSION THREAD

**Discussion by Uzeyir9695** [primefaces/community#655] [PrimeVue]
Is it possible to use single select option or remove checkbox from select option?

**Comment by tugcekucukoglu** [comment]
For which component or for which situations did you ask this question? Can you explain the question in a little more detail so we can understand it?

**Reply by Uzeyir9695** [reply]
For example I want to allow users to select only one option not multiple (which is default set).
Below example allows user to select multiple option using checkboxes but I remove checkboxes from this select-box and when user chooses any option panel should be closed immediately.

<PrimeSelect
        v-model="selectedAircraft"
        :options="allAircraft"
        @change="watchSelectedAircraft"
        optionLabel="name"
        placeholder="Aircraft"
        class="w-full md:w-15rem mt-8 col-md-3 col-sm-12"
        panelClass="z-1"
    ></PrimeSelect>
